### PR TITLE
[ITG-64,ITG-41,ITG-42] Routing and GET task implementation

### PR DESCRIPTION
**Pull request by tgargula**
### Details
* implement `GET /tasks/task_id` endpoint
* add `react-router-dom`
* implement routing
* redirect after clicking task details

### Links
* [ITG-64](https://kulawska.atlassian.net/browse/ITG-64?atlOrigin=eyJpIjoiMjVhMzZiZTI2OGJkNDc0Yzg4NzJhNDI4ZTg0NDFmODIiLCJwIjoiaiJ9)
* [ITG-41](https://kulawska.atlassian.net/browse/ITG-41?atlOrigin=eyJpIjoiODJlNmQ0Nzk0ZjhlNDEwMTg3OTZiNzVjNjcxYjAxYTQiLCJwIjoiaiJ9)
* [ITG-42](https://kulawska.atlassian.net/browse/ITG-42?atlOrigin=eyJpIjoiNWE4ZWNhM2M1ZDlhNDY0YTgyOWVmNTRiNjEzNzA2N2IiLCJwIjoiaiJ9)

### CODE REVIEW

File: bode/bode/resources/tasks/api.py [hunk 1]
```diff
@@ -23,6 +24,13 @@ def post(self, task_data):
 
 @blueprint.route("/<task_id>")
 class TasksById(MethodView):
+    @blueprint.response(200, TaskSchema)
+    def get(self, task_id):
+        try:
+            return Task.query.get(task_id) or abort(404)
+        except DataError:
+            abort(404)
```
Comment (tgargula) [inline bode/bode/resources/tasks/api.py:32]: 1. if a correct UUID is passed -> returns a result if it exists. If not, responds with 404 (line 30)
2. if an incorrect UUID is passed -> `DataError` is caught and also responds with 404 (line 32)

@bkulawska please, test if your implementation of `DELETE` works. I got some errors in an `abort` function when I tried to pass `message` kwarg as you did.

File: cabra/src/pages/Home.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import tw, { styled } from "twin.macro";
+
+import RandomGoat from "./components/RandomGoat";
+import TaskForm from "./components/TaskForm";
+import TasksList from "./components/TasksList";
+
+const Container = styled.div`
+  ${tw`flex flex-col items-center h-screen py-10 overflow-auto max-h-screen`}
+  ${tw`bg-gradient-to-b from-blue-700 to-fuchsia-500`}
+`;
```
Comment (john-sonz) [inline cabra/src/pages/Home.tsx:10]: Looks like this is duplicated across all pages so it'd be nice to extract it into some kind of `Layout` component and use that instead

File: cabra/src/pages/TaskDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  const { data, isLoading } = useQuery(
+    ["task", id],
+    () => axios.get<ITask>(`/tasks/${id}`),
+    { onError: () => navigate("/not-found", { replace: true }), retry: 1 }
+  );
+
+  if (isLoading) return <Container>Loading</Container>;
+
+  return (
+    <>
```
Comment (john-sonz) [inline cabra/src/pages/TaskDetails.tsx:26]: Unnecessary fragment `<>` here

**Review by john-sonz — APPROVED** [review]
:shipit: :shipit: :shipit: :shipit: :shipit:

**Pull request merged**